### ISSUE TITLE
@link tag is not allowed in class comment

### ISSUE THREAD

**Issue by artwist** [squizlabs/PHP_CodeSniffer#695]
I'm confused. Why this code:
```php
<?php

namespace Foo;

/**
 * Sample
 * @link https://github.com/squizlabs/PHP_CodeSniffer
 * @see  https://github.com/squizlabs/PHP_CodeSniffer
 */
class Bar {

}
```
Provides this:
```
----------------------------------------------------------------------
 7 | WARNING | @link tag is not allowed in class comment
   |         | (Squiz.Commenting.ClassComment.TagNotAllowed)
 8 | WARNING | @see tag is not allowed in class comment
   |         | (Squiz.Commenting.ClassComment.TagNotAllowed)
----------------------------------------------------------------------
```

**Comment by aik099** [comment]
I guess this is because that `Squiz.Commenting.ClassComment` sniff checks for them.

**Comment by artwist** [comment]
Yeah, this is obvious. But why these tags are not allowed?
By the way, seems like that sniff deny any tags in class comment:
https://github.com/squizlabs/PHP_CodeSniffer/blob/master/CodeSniffer/Standards/Squiz/Sniffs/Commenting/ClassCommentSniff.php#L110
There are no conditions with tag names.

**Comment by aik099** [comment]
I guess that's rule in that Squiz standard not to use tags in there. If you want to use Squiz standard rules, then you must fix your code to pass them I guess.

**Comment by gsherwood** [comment]
Tags are not allowed just because that's the way the Squiz standard wants it for that code. They are not bad tags. They are just not used within the products that use that standard.

It's highly likely that the restrictions will be relaxed in the future as they've already been relaxed quite a bit over the years.
